src/test-utils/setupTests.ts

```
@@ -337,3 +337,28 @@ vi.mock('../../../services/tokensData', () => ({
 }));
 
 import.meta.env.VITE_PRIVY_APP_ID = 'test';
+
+// Mock SVG imports for Vitest
+// This handles the ReactComponent export from vite-plugin-svgr
+// We need to mock the actual resolved paths that components use
+const React = require('react');
```

Biome (2.1.2)

[error] Shouldn't redeclare 'React'. Consider to delete it or rename it.

'React' is defined here:


(lint/suspicious/noRedeclare)